src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php

```diff
@@ -82,7 +81,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
      */
     public function addEntityFields(FormBuilderInterface $builder, array $options)
     {
-
+        
```

Comment:
Fixed in https://github.com/Guite/MostGenerator/commit/acefde8dab75baa4264fabf5244f8f77a38f1298